### PR TITLE
Add mapping test

### PR DESCRIPTION
**Pull request by mgalang**
Card: https://github.com/wunderio/elasticsearch_helper/projects/1#card-41403383

Added index mapping test

### CODE REVIEW

File: tests/src/Kernel/IndexTest.php [hunk 1]
```diff
@@ -84,19 +108,39 @@ protected function queryIndex($docId) {
...
-    return $response;
+    $this->assertEqual($properties['id']['type'], 'text', 'ID field is found');
+    $this->assertEqual($properties['status']['type'], 'boolean', 'Status field is found');
+    $this->assertEqual($properties['title']['type'], 'text', 'Title field is found');
+    $this->assertEqual(
+      $properties['title']['fields'],
+      ['keyword' => ['type' => 'keyword', 'ignore_above' => 256]],
+      'Title sub-field is found'
+    );
+    $this->assertEqual($properties['uuid']['type'], 'text', 'UUID field is found');
```
Comment (maijs) [inline tests/src/Kernel/IndexTest.php]: I wonder why `id` and `uuid` test positive for `text` type. Fields `id` and `uuid` are supposed to be keywords (https://github.com/wunderio/elasticsearch_helper/blob/8.x-7.x/tests/modules/elasticsearch_helper_test/src/Plugin/ElasticsearchIndex/SimpleNodeIndex.php#L27).

Comment (mgalang) [reply tests/src/Kernel/IndexTest.php]: Good catch. I need to find to setup the mapping on each test. The problem now is that the index is deleted on each test run, so the mapping changes.

**Comment by mgalang** [comment]
This should be ready now.
I created a separate index for mapping testing to avoid conflicts with the index tests.
I also created a functional test for the mapping definition class.

**Review by maijs — APPROVED** [review]
Looks and works good!

**Pull request merged**